scout/server/blueprints/panels/templates/panels/panel_pdf_case_hits.html

```diff
@@ -0,0 +1,76 @@
+{% extends "report_base.html" %}
+
+{% block body %}
+<br>
+<div class="container">
+  <h4>Scout - Institute {{institute.display_name}} - case {{case.display_name}} - Gene panel variant hits</h4> - created on:&nbsp;<strong>{{report_created_at}}</strong><br><br>
```

Comment:
Looks really nice! The title is a little off from the issues intention though. 😊